scripts/build/build-docs.js

```diff
@@ -14,7 +14,8 @@ const parsers = [
   "graphql",
   "postcss",
   "parse5",
-  "markdown"
+  "markdown",
+  "python"
```

Comment:
Shall we do it alphabetically?! Scales better..

Reply:
At the rate we add parsers I _really_ don't think it matter